src/table/table.scss

```diff
@@ -285,7 +285,7 @@
       }
 
       img,
-      svg:not(.iui-radial),
+      .iui-icon,
```

Comment:
Change this back maybe

Reply:
Reverted.

Reply:
I said "maybe" because I'm not sure why the `:not(.iui-radial)` selector is needed here.

With web components, classes will not be applied on the svg but rather on the custom element.